btcclient/client.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2022-2022 The Babylon developers
+// Copyright (c) 2013-2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
```

Comment:
please note which parts of the files in this PR are not copied.

Reply:
Shall I put comments above the non-copied parts?

Reply:
Put comments in the code , or indicate in the review (clicking in this PR "add comment")